src/include/mpir_request.h

```diff
@@ -208,6 +208,9 @@ struct MPIR_Request {
         struct {
             struct MPIR_Debugq *dbg;
         } send;                 /* kind : MPIR_REQUEST_KIND__SEND */
+        struct {
+            struct MPIR_Debugq *dbg;
+        } recv;                 /* kind : MPIR_REQUEST_KIND__RECV */
```

Comment:
Seems ugly with the double `struct` here. Is it necessary?

Reply:
Resolved later. Some squashing might be possible. The commit history so far reflects the development flow.